server/graphql/v2/object/Policies.ts

```diff
@@ -18,6 +27,28 @@ export const Policies = new GraphQLObjectType({
           freeze: { type: GraphQLBoolean },
         }),
       }),
+      resolve(account) {
+        return getPolicy(account, POLICIES.COLLECTIVE_MINIMUM_ADMINS);
+      },
+    },
+    [POLICIES.MAXIMUM_VIRTUAL_CARD_LIMIT_AMOUNT_FOR_INTERVAL]: {
+      name: POLICIES.MAXIMUM_VIRTUAL_CARD_LIMIT_AMOUNT_FOR_INTERVAL,
+      type: new GraphQLObjectType({
+        name: POLICIES.MAXIMUM_VIRTUAL_CARD_LIMIT_AMOUNT_FOR_INTERVAL,
+        fields: () => ({
+          [VirtualCardLimitIntervals.ALL_TIME]: { type: GraphQLInt },
+          [VirtualCardLimitIntervals.DAILY]: { type: GraphQLInt },
+          [VirtualCardLimitIntervals.MONTHLY]: { type: GraphQLInt },
+          [VirtualCardLimitIntervals.PER_AUTHORIZATION]: { type: GraphQLInt },
+          [VirtualCardLimitIntervals.WEEKLY]: { type: GraphQLInt },
+          [VirtualCardLimitIntervals.YEARLY]: { type: GraphQLInt },
```

Comment:
We can keep storing them as integers in the database, but out of consistency the GraphQL API should return them as amounts:

```suggestion
          [VirtualCardLimitIntervals.ALL_TIME]: { type: Amount },
          [VirtualCardLimitIntervals.DAILY]: { type: Amount },
          [VirtualCardLimitIntervals.MONTHLY]: { type: Amount },
          [VirtualCardLimitIntervals.PER_AUTHORIZATION]: { type: Amount },
          [VirtualCardLimitIntervals.WEEKLY]: { type: Amount },
          [VirtualCardLimitIntervals.YEARLY]: { type: Amount },
```